cspell.json

```diff
@@ -205,6 +205,8 @@
     "uuidv",
     "webfonts",
     "useravatar",
-    "rendericon"
+    "rendericon",
+    "Ktps",
+    "Gigstore"
```

Comment:
Rather than adding these here, I think it’s easier to change the test data in our examples to something more generic. e.g. `Route2_name` or adding the cspell ignores to the top of the file.